mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -17,13 +22,39 @@ interface WayPageProps {
  * Way page
  */
 export const WayPage = (props: WayPageProps) => {
+  const navigate = useNavigate();
+  const [wayName, setWayName] = useState<string>("");
+
+  let data: WayPreview;
+
+  /**
+   * Get Way
+   */
+  const loadWay = async () => {
+    data = await WayPreviewDAL.getWayPreview(props.uuid);
+    // Navigate to PageError if transmitted way's uuid is not exist
+    if (!data.uuid) {
```

Comment:
how does it possible that data.uuid is not defined?

Reply:
If way doesnt exist

Reply:
It's possible if I will go to the app by copying link. And if uuid will be copied partly. So validateWrapper will be show that uuid is exist but uuid will be wrong according firebase uuids